Description: Attempt setuptools port
 The original code used distutils as a build system. However, distutils is
 deprecated, and will be removed in a later version of Python. Make sure to
 convert the code to setuptools for future-proofing.
 .
 group-guess (1.2.2-2) unstable; urgency=medium
 .
   * Attempt to port to setuptools for future-proofing.
   * This is a local release for testing.
Author: LandonTheCoder <100165458+LandonTheCoder@users.noreply.github.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: upstream
#Bug: <url in upstream bugtracker>
#Bug-Debian: https://bugs.debian.org/<bugnumber>
#Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: no
#Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2022-05-13

--- group-guess-1.2.2.orig/setup.py
+++ group-guess-1.2.2/setup.py
@@ -1,10 +1,12 @@
 #! /usr/bin/python3
-from distutils.core import setup
+##from distutils.core import setup
+from setuptools import setup
+# Let's hope this works...
 
 #email: 100165458+LandonTheCoder@users.noreply.github.com
 setup(name="Group Guess",
-      version="1.2.1",
-      description="Group Guess, a Family Feud clone",
+      version="1.2.2",
+      description="Game implementing a Family Feud clone",
       author="LandonTheCoder",
       author_email="100165458+LandonTheCoder@users.noreply.github.com",
       url="https://github.com/LandonTheCoder/group-guess/",
